ctapipe/reco/direction_uncertainty.py

```diff
@@ -62,13 +62,13 @@ def predict_subarray_table(self, table):
         table = self.feature_generator(table[quality_valid], subarray=self.subarray)
         X, valid = table_to_X(table, self.features, self.log)
         n_rows = len(table)
-        ang_error = np.full(n_rows, np.nan)
-        ang_error[valid] = self._model.predict(X)
-        ang_error = u.Quantity(ang_error, self.unit, copy=False)
+        dir_uncert = np.full(n_rows, np.nan)
+        dir_uncert[valid] = self._model.predict(X)
+        dir_uncert = u.Quantity(dir_uncert, self.unit, copy=False)
 
         result = Table(
             {
-                f"{self.reconstructor_prefix}_ang_distance_uncert": ang_error,
+                f"{self.reconstructor_prefix}_direction_uncertainty": dir_uncert,
```

Comment:
This column name actually needs to stay the same, as it comes from the definitions in `ctapipe.containers`.

Reply:
could also just rename it to `direction_uncertainty`, since we have not really used that column up to now. It's technically a data model change, but again, that column was never filled, so shouldn't cause a problem.

Reply:
The stereo combiner fills this column when combining alt/az predictions by disp. But I don't think this was used for anything up until now, so changing it shouldn't be a problem (and `direction_uncertainty` is a better name imo).